tests/system/providers/google/cloud/gcs/resources/transform_timespan.py

```diff
@@ -28,11 +28,11 @@
 print(f"Running script, called with source: {source}, destination: {destination}")
 print(f"timespan_start: {timespan_start}, timespan_end: {timespan_end}")
 
-with open(Path(destination) / "output.txt", "w+") as dest:
-    for f in Path(source).glob("**/*"):
+with Path(destination, "output.txt").open("w+") as dest:
+    for f in Path(source).rglob("*"):
         if f.is_dir():
             continue
-    with open(f) as src:
-        lines = [line.upper() for line in src.readlines()]
-        print(lines)
-        dest.writelines(lines)
+        with f.open() as src:
```

Comment:
I think this block should be indented, so it happens for every `f` and not only for the last one. Please review.

Reply:
Yes. Google System tests are not **yet** run regularly. Google team works on automatically running them and providing a dashboard so they would get to this one too.. but we can correct it now.